src/CallFire/Api/DocumentedClient.php

```diff
@@ -0,0 +1,1819 @@
+<?php
```

Comment:
Did that. No pregenerated version and customer will run generation manually when he wants.